opensrp-reveal/src/test/java/org/smartregister/reveal/fragment/SummaryFormsFragmentTest.java

```diff
@@ -0,0 +1,71 @@
+package org.smartregister.reveal.fragment;
+
+import android.app.ProgressDialog;
+import android.support.v7.app.AppCompatActivity;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
+import org.smartregister.reveal.BaseUnitTest;
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.presenter.OtherFormsFragmentPresenter;
+import org.smartregister.reveal.util.LocationUtils;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertNotNull;
+import static org.smartregister.Context.bindtypes;
+
+/**
+ * Created by Richard Kareko on 3/2/20.
+ */
+
+public class SummaryFormsFragmentTest extends BaseUnitTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    @Mock
+    OtherFormsFragmentPresenter presenter;
+
+    @Mock
+    LocationUtils locationUtils;
+
+    @Mock
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5066421)